### ISSUE TITLE
Support aria-labelledby for combobox initialization

### ISSUE THREAD

**Issue by jaredcunha** [uswds/uswds#4114]
**Is your feature request related to a problem? Please describe.**
Combo box will not build unless there is a `<label>` associated with it. 

**Describe the solution you'd like**
The script should build if the combo box has _either_ a `<label>` associated, or the underlying `<select>` includes an `aria-labelledby`.

**Additional context**
<img width="1037" alt="Screen Shot 2021-03-19 at 5 58 15 PM" src="https://user-images.githubusercontent.com/1094951/111846210-b1141c80-88dc-11eb-8dc9-c825083a72c4.png">
See this codepen example: https://codepen.io/jaredcunha/pen/VwmEXbG

Form fields in this table can use use `aria-labelledby` to associate to the column header when entering tabular data. The combobox is not supported in this use case.

**Comment by annepetersen** [comment]
Hi there: this is one of many submitted PRs/issues that have languished here. USWDS is currently understaffed, so I'm closing items that aren’t as high on the priority list right now, including this one, unfortunately.

**Comment by chandracarney** [comment]
Though this is closed, here are some discussions we had in June, though it needs to be verified / explored more
- Idea: maybe fails to build because it doesn't have a label element associated
- Could be expanded to aria-label so it builds an accessible name

Resolving this issue will require working with an accessibility testing subject matter expert who can test with necessary physical devices and tools.
